ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomCommandingDemoController.swift

```diff
@@ -10,17 +10,17 @@ class BottomCommandingDemoController: UIViewController {
     override func loadView() {
         view = UIView()
 
-        let optionTableView = UITableView(frame: .zero, style: .plain)
+        let optionTableViewController = UITableViewController(style: .plain)
+        let optionTableView: UITableView = optionTableViewController.tableView
```

Comment:
Can the type here be inferred?

Reply:
The type has to be explicit for the implicitly unwrapped optional to work (UITableViewController.tableView is of type TableView! <- including that "!").